.github/workflows/ci.yml

```diff
@@ -2,10 +2,11 @@ name: Garage CI
 
 env:
   GARAGE_GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  TRAVIS_COMMIT_RANGE: origin/${{ github.base_ref }}...origin/${{ github.head_ref }}
-  DOCKER_TAG: garage-ci-${{ github.run_id }}
+  COMMIT_RANGE: origin/${{ github.base_ref }}...origin/${{ github.head_ref }}
+  DOCKER_TAG: garage-ci-189158592
```

Comment:
this tag doesn't exist anymore. See here: https://github.com/rlworkgroup/garage/packages